lib/ts/recipe/emailverification/emailVerificationClaim.ts

```diff
@@ -2,6 +2,10 @@ import { DateProviderReference } from "supertokens-website/utils/dateProvider";
 import { SessionClaimValidator, BooleanClaim } from "../session";
 import { RecipeInterface } from "./types";
 
+function getThresholdAwareDefaultValue(defaultVal: number) {
+    return Math.max(defaultVal, DateProviderReference.getReferenceOrThrow().dateProvider.getThresholdInSeconds());
```

Comment:
should we add +1 to the `DateProviderReference.getReferenceOrThrow().dateProvider.getThresholdInSeconds()`? If not, then why not?

Reply:
Instead of adding 1, we instead modify the `setClientClockSkewInMillis` function in date provider to allow the clock skew values equal to the threshold. 

It means that if the threshold is 10 and clock skew is also 10, then DateProvider will account for it and there should be no issues using age values equal to the threshold.

Reply:
Everywhere we check if the maxAge is OK, we are checking if it is less than the threshold, so I think it also makes sense here: if I set the threshold to 30 seconds, it makes sense that the default `refetchTimeOnFalse` is set to 30 seconds as well (while `maxAgeInSeconds` remains unchanged). Adding an extra second seems random, since I could manually set `refetchTimeOnFalseInSeconds = 30.5`.